content/sensu-go/6.3/sensuctl/create-manage-resources.md

```diff
@@ -244,10 +245,10 @@ Requests to update agent-managed entities via sensuctl will fail and return an e
 |sensuctl edit types |   |   |   |
 --------------------|---|---|---|
 `asset` | `check` | `cluster` | `cluster-role`
-`cluster-role-binding` | `entity` | `event` | `filter`
-`handler` | `hook` | `mutator` | `namespace`
-`role` | `role-binding` | `silenced` | `user`
-[`auth`][26] | | |
+`cluster-role-binding` | `component` | `entity` | `event`
+`filter` | `handler` | `hook` | `mutator`
+`namespace` | `role` | `role-binding` | `rule-template`
+`silenced` | `user` | [`auth`][26] | | 
```

Comment:
I'm not sure if lower-case "component" and lower-case and hyphenated "rule-template" is correct here for sensuctl edit. Just guessing -- please confirm

Reply:
I don't think there are "shortcuts" defined for them, so one needs to use the fully qualified name as listed by `sensuctl describe-type all`: `bsmv1.RuleTemplate` and `bsmv1.ServiceComponent`.

Reply:
Where can I look in the code to find the resource names to use with difference sensuctl commands? None of the rule template or service component permutations I've tried seem to work. I think more generally these sensuctl lists are in some disarray and I'd like to fix them. Or is there a sensuctl command that lists the resource names to use with each command type, kind of like the describe-type list but for create, edit, etc?

Reply:
Disregard -- see https://sensu.slack.com/archives/C60EEQFH8/p1616508141272600